client/my-sites/plans/main.jsx

```diff
@@ -139,8 +142,15 @@ const Plans = React.createClass( {
 
 	render() {
 		const selectedSite = this.props.sites.getSelectedSite();
+		let PlanList = OldPlanList;
 		let hasJpphpBundle,
-			currentPlan;
+			currentPlan,
+			mainStyle;
+
+		if ( showPlanFeatures ) {
+			PlanList = PlanFeaturesList;
```

Comment:
PlanList and PlanFeaturesList will probably require different prop values. I think it'd be better to be explicit about what properties each component needs, even if it is more verbose.

So below, maybe something like:
```
{ showPlanFeatures ? <PlanFeaturesList /> : <PlanList> }
```